data/_models/shared/Catalog.csv

```diff
@@ -0,0 +1,11 @@
+tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles
```

Comment:
for this one we need to check with data catalogue model on the 'how' because there is also catalog table.
More importantly, I believe that in many cases DCAT should go via annotations of other data models instead of having a seperate data model.